lib/src/Uncertainty/Algorithm/Transformation/ProcessTransformation/BoxCoxFactory.cxx

```diff
@@ -208,6 +213,134 @@ class BoxCoxGLMOptimization : public EvaluationImplementation
   Scalar sumLog_;
 };
 
+class BoxCoxLMOptimization : public EvaluationImplementation
+{
+
+public:
+  BoxCoxLMOptimization(const Sample &inputSample,
+                       const Sample &shiftedOutputSample,
+                       const Basis &basis)
+      : inputSample_(inputSample)
+      , shiftedOutputSample_(shiftedOutputSample)
+      , basis_(basis)
+      , sumLog_(0.0)
+      , algo_()
+  {
+    initialize();
+    computeSumLog();
+  }
+
+  BoxCoxLMOptimization(const Sample &inputSample,
+                       const Sample &shiftedOutputSample)
+      : inputSample_(inputSample)
+      , shiftedOutputSample_(shiftedOutputSample)
+      , basis_()
+      , sumLog_(0.0)
```

Comment:
now with c++11 you can initialize attributes at declaration

Reply:
ok but to have the "same behaviour" as other classes I will probably keep it like this